lang_tests/run.rs

```diff
@@ -0,0 +1,31 @@
+use std::{fs::read_to_string, process::Command};
+
+use lang_tester::LangTester;
+
+const COMMENT: &str = "//";
+
+fn main() {
+        LangTester::new()
+            .test_dir("lang_tests")
+            .test_file_filter(|p| {
+                p.extension().map(|x| x.to_str().unwrap()) == Some("c")
+            })
+            .test_extract(move |p| {
+                read_to_string(p)
+                    .unwrap()
+                    .lines()
+                    .skip_while(|l| !l.starts_with(COMMENT))
+                    .take_while(|l| l.starts_with(COMMENT))
+                    .map(|l| &l[COMMENT.len()..])
+                    .collect::<Vec<_>>()
+                    .join("\n")
+            })
+            .test_cmds(move |p| {
+                    let mut clang = Command::new("clang");
+                    clang.args(&["-emit-llvm", "-o", p.to_str().unwrap().replace(".c",".bc").as_str(), "-c", p.to_str().unwrap()]);
+                    let mut bc = Command::new("cargo");
+                    bc.args(&["run", "--release", p.to_str().unwrap().replace(".c",".bc").as_str()]);
+                    vec![("clang", clang), ("bc", bc)]

```

Comment:
Let's name these "Compiler" and "Runtime" like in our other projects. Or if you prefer "Compiler" and "Interpreter".